environment-variables.js

```diff
@@ -1,7 +1,32 @@
 const environmentVariables = {
   development: {
-    BASE_URL: 'http://192.168.1.13:8080/api',
-    OAUTH_URL: 'http://192.168.1.13:8080/oauth/token',
+    BASE_URL: 'http://localhost:8080/api',
+    OAUTH_URL: 'http://localhost:8080/oauth/token',
+    USERS_LIST_URL: 'http://localhost:8080/api/v1/users',
+    //users
+    GET_USER_URL: 'http://localhost:8080/api/v1/users/1',
```

Comment:
Do we want to keep the actual Id out of environment variables? Seems like this file shouldn't know about the data being tested. I.E. change `http://localhost:8080/api/v1/users/1` to `http://localhost:8080/api/v1/users/` and then in the actual places they're used, do the `${GET_USER_URL}/${userId}`.